src/content/docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-activemq.mdx

```diff
@@ -0,0 +1,183 @@
+
+1. Go to file path `/etc/newrelic-infra/integrations.d/jvm-metrics.yml`
+2. Add this snippet:
+
+```yml
+collect:
+  - domain: java.lang
+  event_type: JVMSampleActiveMQ
+  beans:
+    - query: type=GarbageCollector,name=*
+      attributes:
+        - CollectionCount
+        - CollectionTime
+    -query: type=Memory
+      attributes:
+        - HeapMemoryUsage.Committed
+        - HeapMemoryUsage.Init
+        - HeapMemoryUsage.Max
+        - HeapMemoryUsage.Used
+        - NonHeapMemoryUsage.Committed
+        - NonHeapMemoryUsage.Init
+        - NonHeapMemoryUsage.Max
+        - NonHeapMemoryUsage.Used
+    - query: type=Threading
+      attributes:
+        - ThreadCount
+        - TotalStartedThreadCount
+    - query: type=ClassLoading
+      attributes:
+        - LoadedClassCount
+    - query: type=Compilation
+      attributes:
+        - TotalCompilationTime
+  - domain: org.apache.activemq 
+    event_type: JVMSampleActiveMQ 
+    beans: 
+      - query: type=Broker,brokerName=* 
+        attributes: 
+          - AverageMessageSize 
+          - Brokerid 
+          - BrokerName 
+          - BrokerVersion 
+          - CurrentConnectionscount 
+          - DataDirectory 
+          - DurableTopicSubscribers 
+          - DynamicDestinationProducers 
+          - InactiveDurable 
+          - Topic 
+          - Subscribers 
+          - JMS 
+          - JobScheduler 
+          - JobschedulerStorelimit 
+          - JobSchedulerStorePercentUsage 
+          - MaxMessageSize 
+          - Memorylimit 
+          - MemoryPercentUsage 
+          - MinMessageSize 
+          - Persistent 
+          - Queue 
+          - Producers 
+          - QueueSubscribers - Queues - slave - StatisticsEnabled - StoreLimit - StorePercentUsage 
+          - Templimit 
+          - TempPercentUsage 
+          - TemporaryQueue 
+          - Producers 
+          - TemporaryQueueSubscribers 
+          - TemporaryQueues 
+          - TemporaryTopicProducers 
+          - TemporaryTopicSubscribers 
+          - TemporaryTopics 
+          - TopicProducers 
+          - TopicSubscribers 
+          - Topics 
+          - TotalConnectionsCount 
+          - TotalConsumerCount 
+          - TotalDequeueCount 
+          - TotalEnqueueCount 
+          - TotalMessageCount 
+          - TotalProducerCount 
+          - TransportConnectors 
+          - Uptime 
+          - UptimeMills
+```
+  </Collapser>
+  <Collapser
+    id="step3"
+    title="Step 3. Forward your ActiveMQ logs"
+  >
+With your monitoring environment set up, forward your ActiveMQ logs to New Relic. To forward logs:
+
+1. Go to your ActiveMQ `logging.yml` file.
+2. Add this snippet anywhere to the file:
+
+```yml
+- name: activemq.log
+  file: /opt/activemq/data/activemq.log
+  attributes:
+    logtype: activemq-log
+- name: activemq_audit.log
+  file: /opt/activemq/data/activemq.log
+  attributes:
+    logtype: activemq-audit-log
```

Comment:
Hi @akristen , requesting you to kindly remove the lines from 167 to 170
Thank you 😊